test/src/test_infra.cpp

```diff
@@ -779,8 +767,6 @@ void report_err2_fp64(void (*test_func)(size_t, const double *, const double *,
   free(x);
```

Comment:
If you are adding googletests, is it worth to add in an assert on the max_ulp_err?

Reply:
Many tests are failing, so I'll let @PingTakPeterTang figure out where it is best to add the assert.